src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
```

Comment:
According to the checklist, don't use abstract classes to set behavior for classes if it's not necessary. Ensure that using an abstract class here is justified and that an interface wouldn't be a better choice.